src/pages/AssetMapPage/AMForceGraph.tsx

```diff
@@ -54,19 +55,16 @@ const nodeBorderScaler = (f: number) => 0.7 + f * 2.5;
 export const AMForceGraph = () => {
   const classes = useStyles();
   const fgRef = useRef<any>(null);
-  const recoilMapGraphData = useMapGraphData();
+  const circleId = useSelectedCircleId();
```

Comment:
Rather than fetchign the CircleId from recoil, could we fetch it from the URL like `const { circleId } = useParams();`

https://reactrouter.com/en/6.5.0/hooks/use-params#useparams